fades/helpers.py

```diff
@@ -208,3 +210,37 @@ def check_pypi_updates(dependencies):
 
     dependencies["pypi"] = dependencies_up_to_date
     return dependencies
+
+
+def _pypi_head_package(pkg_name):
+    """Hit pypi with a http HEAD to check if pkg_name exists."""
+    url = BASE_PYPI_URL.format(name=pkg_name)
+    req = request.Request(url, method='HEAD')
+    try:
+        response = request.urlopen(req)
+    except HTTPError as http_error:
+        if http_error.code == HTTP_STATUS_NOT_FOUND:
+            return False
+        else:
+            raise http_error
+    else:
```

Comment:
considering that in the "except" never goes through (or it returns or it raises an exception), I'll kill this else and deindent the code inside it.

Reply:
done